zebra/rt_netlink.c

```diff
@@ -652,9 +661,13 @@ int netlink_route_change(struct sockaddr_nl *snl, struct nlmsghdr *h,
 			 ns_id_t ns_id, int startup)
 {
 	int len;
-	vrf_id_t vrf_id = ns_id;
+	vrf_id_t vrf_id;
```

Comment:
Maybe we can just remove this `vrf_id` variable from here, since we can't know the VRF ID before we parse the `RTA_TABLE` netlink attribute later. We're basically logging wrong information.